src/v2_layout/index.ts

```diff
@@ -0,0 +1,13 @@
+import { V2_ColDiv } from "./col-div";
+import { V2_Container } from "./container";
+import { V2_Content } from "./content";
+import { V2_Section } from "./section";
+
+export const v2_Layout = {
+    V2_Section: V2_Section,
+    V2_Container: V2_Container,
+    V2_Content: V2_Content,
+    V2_ColDiv: V2_ColDiv,
```

Comment:
after looking through the code, realised that the components are typically used as `Layout.Section` etc., so it's fine to retain the sub-component names

`Layout` will still be renamed to `V2_Layout`

```suggestion
    Section,
    Container,
    Content,
    ColDiv,
```